components/mpas-ocean/src/Registry.xml

```diff
@@ -3527,7 +3570,10 @@
 			 description="Divergence of transmission through interfaces of surface fluxes below the surface layer at cell centers. These are applied only to river runoff."
 			 packages="forwardMode;analysisMode"
 		/>
-
+		<var name="fractionAbsorbedSubglacialRunoff" type="real" dimensions="nVertLevels nCells Time" units="fractional"
+			 description="Divergence of transmission through interfaces of surface fluxes below the surface layer at cell centers. These are applied only to subglacial runoff."
+			 packages="dataSubglacialRunoffFluxPKG"
+		/>
```

Comment:
```suggestion
		/>

```
Maybe add back the blank line that was here before, just for visual separation between fields for different purposes?

Reply:
Done